src/app/child-dev-project/attendance/activity-setup/activity-setup.component.html

```diff
@@ -0,0 +1,43 @@
+<div fxLayout="column" class="overall-container">
+  <div fxFlex>
+    <mat-form-field>
+      <mat-label>Record event for</mat-label>
+      <input matInput [(ngModel)]="date" required
+             [matDatepicker]="datePicker"
+             (dateChange)="setNewDate($event.value)"
+      >
+      <mat-datepicker-toggle matSuffix [for]="datePicker"></mat-datepicker-toggle>
+      <mat-datepicker #datePicker></mat-datepicker>
+    </mat-form-field>
+  </div>
+
+  <div class="flex-scroll-container" fxFlex="grow">
+    <div fxLayout="row wrap" fxLayoutGap="8px">
+      <app-activity-card *ngFor="let event of existingEvents"
+                         [event]="event"
+                         (click)="selectedEvent = event"
+                         [highlighted]="selectedEvent === event"
+                         class="event-card">
+      </app-activity-card>
+    </div>
+
+    <div fxLayoutGap="8px">
+      <button mat-stroked-button [disabled]="visibleActivities.length === allActivities.length">Show more</button>
+      <button mat-button>My event is not listed ...</button>
+    </div>
+  </div>
+
+  <div class="mat-elevation-z5 bottom-control" fxFlex="100px">
+    <div fxLayout="row" fxLayoutGap="16px">
+      <button mat-raised-button color="primary" class="confirm-button"
+              [disabled]="!selectedEvent"
+              (click)="eventSelected.emit(selectedEvent)">
+        Start Roll Call
+      </button>
+
+      <app-activity-card *ngIf="selectedEvent" [event]="selectedEvent"
+                         class="mat-elevation-z0 selected-card" fxFlex>
+      </app-activity-card>
```

Comment:
The challenge is that the cards can have different height at the moment, depending on line breaks and what information is available. I kept it like this because I felt it more important to keep the bottom panel at a fixed size while not "wasting" too much valuable screen space. Not ideal, I agree, but the best thing I could come up with till now.